desub-legacy/src/substrate_types.rs

```diff
@@ -33,8 +33,90 @@ use std::{convert::TryFrom, fmt};
 pub use self::data::Data;
 
 pub type Address = MultiAddress<AccountId32, u32>;
-pub type Vote = pallet_democracy::Vote;
-pub type Conviction = pallet_democracy::Conviction;
+
+/// Stripped down version of https://docs.substrate.io/rustdocs/latest/pallet_democracy
+/// Remove when/if the real pallet_democracy is published.
```

Comment:
maybe open an tracking issue for this or something?!

Reply:
Nah, it's like completely up in the air if the frame pallets will ever be published.